### DISCUSSION TITLE
How we can update Unbound in Pi-Hole container?

### DISCUSSION THREAD

**Discussion by psychedelicu** [tteck/Proxmox#3229] [General] [answered]
Hello! How we can update Unbound in Pi-Hole container? because the "update" command is not updating it, and the current version is 1.20.0.

![image](https://github.com/tteck/Proxmox/assets/35727400/f6f55ad4-7400-48a4-bc5c-367b3aaf1c93)

**Comment by tteck** [comment, marked as answer]
Unbound is updated via `apt`
The current version in the Debian repository is 1.17.1
```
apt-cache policy unbound
unbound:
  Installed: 1.17.1-2+deb12u2
  Candidate: 1.17.1-2+deb12u2
  Version table:
 *** 1.17.1-2+deb12u2 500
        500 http://security.debian.org bookworm-security/main amd64 Packages
        100 /var/lib/dpkg/status
     1.17.1-2+deb12u1 500
        500 http://deb.debian.org/debian bookworm/main amd64 Packages
```

**Reply by psychedelicu** [reply]
There are many new releases since 1.17.1. Why they don't update it?
